frame/staking/src/lib.rs

```diff
@@ -761,9 +760,6 @@ decl_storage! {
 
 		/// The earliest era for which we have a pending, unapplied slash.
 		EarliestUnappliedSlash: Option<EraIndex>;
-
-		/// The version of storage for upgrade.
-		StorageVersion: u32;
```

Comment:
I suspect the storage version hasn't been removed from storage in kusama

Reply:
But if I query on kusama the key `twox128("Staking") ++ twox128("StorageVersion") == 0x5f3e4907f716ac89b6347d15ececedca308ce9615de0775a82f8a94dc3d285a1` then I get no value, and I can't understand why yet.